streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.java

```diff
@@ -61,12 +62,13 @@
  *
  * This class do the translation between generic {@link Options} into {@link DBOptions} and {@link ColumnFamilyOptions}.
  */
-public class RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter extends Options {
+class RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter extends Options {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.class);
```

Comment:
Use `log` as the name of Logger instance here to be consistent with logger usage in general.